safe-global/safe-homepage
# feat: Add ecosystem page

## What it solves

Part of #161 

This adds a first version of the ecosystem page. There are a few missing static elements that will be delivered once the [design](https://www.figma.com/file/doceHL65jwlEbAGegLRnq2/Landing-Page?node-id=5134-12535&t=qeg1KJl8zkgpMFqW-0) is ready.

## How to test

1. Open `/ecosystem`
2. Observe the ecosystem projects

## Screenshots

<img width="1451" alt="Screenshot 2023-05-08 at 16 33 33" src="https://user-images.githubusercontent.com/5880855/236852083-a810b7e7-5040-41c2-8fd0-6065a6fbd46c.png">
<img width="1257" alt="Screenshot 2023-05-08 at 16 33 42" src="https://user-images.githubusercontent.com/5880855/236852113-abeda498-ffd8-4d62-8179-19175eba3c0c.png">
